src/libraries/Common/src/Interop/Unix/System.Native/Interop.ReadStdinUnbuffered.cs

```diff
@@ -13,7 +13,7 @@ internal static partial class Sys
         internal static extern unsafe int ReadStdin(byte* buffer, int bufferSize);
 
         [DllImport(Libraries.SystemNative, EntryPoint = "SystemNative_InitializeConsoleBeforeRead")]
-        internal static extern unsafe void InitializeConsoleBeforeRead(byte minChars = 1, byte decisecondsTimeout = 0);
+        internal static extern unsafe void InitializeConsoleBeforeRead(bool enableCrNl, byte minChars = 1, byte decisecondsTimeout = 0);
```

Comment:
nit: rename `enableCrNl` to `convertCrToNl`.
nit2: with how the code ended up, `= false` may be a nice default.